openjdk8/src/main/docker/setup-env.d/05-utils-env.bash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Small nit: I would probably just name this file `05-utils.bash`. Nothing in this file directly has to do with the environment.